### ISSUE TITLE
App crashes when adding multiple pilots, then going back to edit upgrades on the first pilot

### ISSUE THREAD

**Issue by randy-ja** [timrisi/AuroraSquadBuilder#28]
Steps to reproduce 
Create standard 100 pt Rebel squadron
Add VCX-100 pilot and upgrades (include ghost title)
Add Attack shuttle pilot and Phantom title
Add 3rd ship and upgrades
Go back to ghost and try to change an upgrade
App crashes

I’m using an iPhone 6 on iOS 11.0.2.

**Comment by timrisi** [comment]
Hey, this is the second crash I've been able to reproduce now!  (Finally just got another crash I could repro today.  Before that I couldn't repro any of the ones I'd had reported).  I'm working on tracking down the problem now.

**Comment by timrisi** [comment]
Turns out the crash happens pretty much any time you add multiple ships in a squadron then go back and start adding upgrades to the first one.  It seems to be an issue in Xamarin.Forms somewhere (happens after updating that's version, stack trace shows all XF code), so haven't yet found a way to fix it or a workaround.

**Comment by timrisi** [comment]
I tracked down when the crash is happening.  Based on the stack trace, it appears to be happening in Xamarin.Forms code.  I tested this by loading up the code from 1.3.1 and changing just the version of Xamarin.Forms being used.  As soon as I update to the latest version of XF, I get the same crash, without any other changes from 1.3.2, indicating that the problem is not caused by any of my other changes.  The version update is needed in order to support the iPhone X screen.  I created a thread on the XF forums asking for assistence and am hoping for a fix or workaround.  https://forums.xamarin.com/discussion/120074/crashing-in-xamarin-forms-platform-ios-labelrenderer-updatetext#latest.  Will keep looking to see if there is something else I can do in the meantime.
